aquality-tracking-integrations-core/src/main/java/aquality/tracking/integrations/core/FinalResultId.java

```diff
@@ -0,0 +1,12 @@
+package aquality.tracking.integrations.core;
+
+public class FinalResultId {
+
+    public static final int FAILED = 1;
```

Comment:
what does 4 mean?)

Reply:
Have no idea. Looks like these are only available.

> You should specify 'final_result_id' - Failed: 1, Passed: 2, Not Executed: 3, Pending: 5.

Reply:
4 is "IN_PROGRESS" , but it is not  actual for test results

Reply:
In this case it can be skipped.